manifest.ini

```diff
@@ -17,6 +17,7 @@ account_server_url = https://api.accounts.firefox.com/v1
 oauth_server_url = https://oauth.accounts.firefox.com/v1
```

Comment:
looks like a lot of the urls in the manifest use a base url.  you might want to consider creating "base_url" for each test env and then many of the paths can probably be defined as DEFAULT for any test env.  also, when eventually Ops passes in the host URL dynamically, you'll just need to change the base url and not every single one of the paths.  since this shouldn't block your new test from landing, i'll file an issue as an enhancement.